flytectl/pkg/ext/launch_plan_fetcher.go

```diff
@@ -26,8 +26,13 @@ func (a *AdminFetcherExtClient) FetchAllVerOfLP(ctx context.Context, lpName, pro
 }
 
 // FetchLPLatestVersion fetches latest version for give launch plan name
-func (a *AdminFetcherExtClient) FetchLPLatestVersion(ctx context.Context, name, project, domain string, filter filters.Filters) (*admin.LaunchPlan, error) {
-	// Fetch the latest version of the task.
+func (a *AdminFetcherExtClient) FetchLPLatestVersion(ctx context.Context, name, project, domain string) (*admin.LaunchPlan, error) {
+	// Fetch the latest version of the LaunchPLan.
+	filter := filters.Filters{
+		SortBy: "created_at",
+		Limit:  1,
+		Asc:    false,
```

Comment:
I'm not entirely sure if I need to specify `Asc` but better safe than sorry.

Reply:
better be explicit yeah